diff --git a/node_modules/@remix-run/react/dist/esm/single-fetch.js b/node_modules/@remix-run/react/dist/esm/single-fetch.js
index a7e905e..49650a4 100644
--- a/node_modules/@remix-run/react/dist/esm/single-fetch.js
+++ b/node_modules/@remix-run/react/dist/esm/single-fetch.js
@@ -210,21 +210,13 @@ function singleFetchUrl(reqUrl) {
   return url;
 }
 async function fetchAndDecode(url, init) {
-  var _res$headers$get;
   let res = await fetch(url, init);
-  if ((_res$headers$get = res.headers.get("Content-Type")) !== null && _res$headers$get !== void 0 && _res$headers$get.includes("text/x-turbo")) {
-    invariant(res.body, "No response body to decode");
-    let decoded = await decodeViaTurboStream(res.body, window);
-    return {
-      status: res.status,
-      data: decoded.value
-    };
-  }
-
-  // If we didn't get back a turbo-stream response, then we never reached the
-  // Remix server and likely this is a network error - just expose up the
-  // response body as an Error
-  throw new Error(await res.text());
+  invariant(res.body, "No response body to decode");
+  let decoded = await decodeViaTurboStream(res.body, window);
+  return {
+    status: res.status,
+    data: decoded.value
+  };
 }
 
 // Note: If you change this function please change the corresponding
diff --git a/node_modules/@remix-run/react/dist/single-fetch.js b/node_modules/@remix-run/react/dist/single-fetch.js
index 0946c2b..61cac19 100644
--- a/node_modules/@remix-run/react/dist/single-fetch.js
+++ b/node_modules/@remix-run/react/dist/single-fetch.js
@@ -234,21 +234,13 @@ function singleFetchUrl(reqUrl) {
   return url;
 }
 async function fetchAndDecode(url, init) {
-  var _res$headers$get;
   let res = await fetch(url, init);
-  if ((_res$headers$get = res.headers.get("Content-Type")) !== null && _res$headers$get !== void 0 && _res$headers$get.includes("text/x-turbo")) {
-    invariant(res.body, "No response body to decode");
-    let decoded = await decodeViaTurboStream(res.body, window);
-    return {
-      status: res.status,
-      data: decoded.value
-    };
-  }
-
-  // If we didn't get back a turbo-stream response, then we never reached the
-  // Remix server and likely this is a network error - just expose up the
-  // response body as an Error
-  throw new Error(await res.text());
+  invariant(res.body, "No response body to decode");
+  let decoded = await decodeViaTurboStream(res.body, window);
+  return {
+    status: res.status,
+    data: decoded.value
+  };
 }
 
 // Note: If you change this function please change the corresponding
